airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistryV0.py

```diff
@@ -38,18 +34,8 @@ class Config:
 
 
 class JobType(BaseModel):
-    __root__: Literal[
-        "get_spec",
```

Comment:
I have a feeling CI will get mad since it looks like these were not run through the formatter.

I think you can run 

```
airbyte-ci format fix python
```

unless something has changed in the last 3 months.

Reply:
Done, It didn't changed any file: 
<img width="1149" alt="image" src="https://github.com/airbytehq/airbyte/assets/6792130/b3b54901-18dc-4652-8f9a-2ce4997de72f">